README_new.md

```diff
@@ -39,23 +39,25 @@ These are the main areas we in the iOS team will focus our assessment of candida
 
 ### The iOS recruitment process
 
-Our recruitment process is different for candidates who are interested in working remotely compared to the process for candidates who are interested in working on-site at our office in London:
- - on-site: Initial call -> Demo -> 3-hour face-to-face technical interview
- - remote: Initial call -> Demo -> one or more Skype interviews -> 1 working day with us (at our expense)
+Our recruitment process is different between candidates who are interested in working remotely and candidates who are interested in working on-site at our office in London, in **two** ways:
+- remote candidates are *required* to be proficient in **reactive programming** (`Reactive Swift`/`Reactive Cocoa` or `rxSwift`) at an above-average level
```

Comment:
`RxSwift`

Reply:
Fixed in https://github.com/Babylonpartners/iOS-Interview-Demo/pull/4/commits/06df7f77de32481bd90c478a250b0e77b28d3898.